bundles/com.salesforce.bazel.eclipse.core/src/com/salesforce/bazel/eclipse/core/classpath/BazelClasspathContainer.java

```diff
@@ -78,4 +90,13 @@ public int getKind() {
     public IPath getPath() {
         return path;
     }
+
+    /**
+     * A array of unloaded classpath entries.
+     *
+     * @return
+     */
+    public IClasspathEntry[] getUnloadedEntries() {
```

Comment:
Can we rename to `getTransitiveEntries`? I think that better communicates the purpose.